OptimoveSDK/Sources/Classes/Optimove.swift

```diff
@@ -61,27 +62,41 @@ typealias Logger = OptimoveCore.Logger
                     throw GuardError.custom("Failed on OptimobileSDK initialization. Reason: \(error.localizedDescription)")
                 }
             }
+        }   
+
+        if config.isPreferenceCenterConfigured() {
+            shared.container.resolve { serviceLocator in
+                do {
+                    try OptimovePreferenceCenter.initialize(config: config, storage: serviceLocator.storage(), networkClient: NetworkClientImpl())
```

Comment:
this uses storage to get customerID. but when is customerID set / which features are required for it to be set?

i think optimove is needed. e.g. check this condition in android